xbmc/interfaces/legacy/ListItem.h

```diff
@@ -213,6 +213,58 @@ namespace XBMCAddon
       void setLabel2(const String& label);
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_xbmcgui_listitem
+      /// @brief \python_func{ getDateTime() }
+      ///-----------------------------------------------------------------------
+      /// Returns the listitem datetime (ISO 8601).
+      ///
+      /// @return                   string or unicode - datetime string (ISO 8601).
```

Comment:
Technically, we use W3C time, which is a subset of ISO 8601 [specifically, ISO 8601 : 1988 (E)]. From my research:

It turns out ISO 8601 is a big standard. To simplify things, the W3C (in charge of the web) decided to use a strict subset of ISO 8601. The difference is that only three formats are allowed:

* 2020-09-17T14:09:05 (timezone-naive)
* 2020-09-17T14:09:05Z (timezone aware, Zulu timezone == no offset from UTC)
* 2020-09-17T14:09:05+0000 (timezone aware, numeric offset from UTC)

Reply:
You're right. I've clarified the format in the documentation.